joplin/templates/wagtailadmin/pages/create.html

```diff
@@ -0,0 +1,45 @@
+{% extends "wagtailadmin/pages/create.html" %}
+
+{% load wagtailadmin_tags %}
+{% load i18n %}
+{% load l10n %}
+
+{% block extra_js %}
+    {{ block.super }}
+    {% include "wagtailadmin/pages/_editor_js.html" %}
+
+    {% comment %}
+        Additional js from widgets media. Allows for custom widgets in admin panel.
+    {% endcomment %}
+    {{ edit_handler.form.media.js }}
+
+    {% comment %}
+        Additional HTML code that edit handlers define through 'html_declarations'. (Technically this isn't Javascript, but it will generally be data that exists for Javascript to work with...)
+    {% endcomment %}
+    {{ edit_handler.html_declarations }}
+
+    <script>
+        $(function(){
+            $('#page-edit-form .tab-content section.active input').first().trigger('focus');
+            /* Make user confirm before leaving the editor if there are unsaved changes */
+            {% trans "This page has unsaved changes." as confirmation_message %}
+            enableDirtyFormCheck(
```

Comment:
this wasn't already there?

Reply:
no, this file extends the `joplin/templates/wagtailadmin/pages/create.html` file from Wagtail core

Reply:
hmm, extending generally means you still have the same page there but are replacing some blocks, where as if you don't use the extend tag then you can replace the entire page. 

I'd think the block.super should be able to get enableDirtyFormCheck in there.

Reply:
thanks for explaining. fixed in the next branch